.github/workflows/build.yml

```diff
@@ -20,21 +20,25 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["8.0", "8.1"]
-                symfony: ["^5.4"]
-                sylius: ["~1.11.2", "~1.12.0"]
-                node: ["14.x"]
+                symfony: ["^5.4", "^6.0"]
+                sylius: ["~1.11.4", "~1.12.0"]
+                node: ["16.x"]
                 mysql: ["5.7", "8.0"]
                 pdf: [true]
 
                 include:
                     -
                         php: "8.1"
-                        symfony: "^5.4"
-                        sylius: "~1.11.2"
-                        node: "14.x"
+                        symfony: "^6.0"
+                        sylius: "~1.12.0"
+                        node: "16.x"
                         mysql: "8.0"
                         pdf: false
```

Comment:
Would be nice to add two more, one `sylius 1.12/symfony 5.4` and the one that was here previously

Reply:
The initial idea with `pdf: false` was to test it only once on the highest versions but we support two Sylius versions now. Therefore I'll add an extra one with Sylius 1.11 and SF 5.4 :+1: